requests/add_spyder_base.yml

```
@@ -0,0 +1,6 @@
+action: add_feedstock_output
+feedstock_to_output_mapping:
+  # this entry adds a package name
+  - spyder: 
+    - spyder-base
+    - spyder
```

```suggestion
  - spyder: spyder-base
```

seems like we only must add the "additional" packages, not all packages that already exist.